packages/xod-client/src/workers/wasm.worker.js

```diff
@@ -0,0 +1,46 @@
+// Worker imported as usual js file (using `worker-loader`)
+// so tests fails with ReferenceError "self is not defined"
+// to avoid this we define _self as self for worker
+// or empty object for tests and etc.
```

Comment:
:scroll: This file is a WASM web worker implementation.
The worker is imported as a usual JS module by `worker-loader`.
In this case tests fail with ReferenceError "self is not defined".
To avoid this we define _self as self for worker, or an empty object
for tests and etc.